src/bindings/js/node/package.json

```diff
@@ -18,7 +18,8 @@
     "build": "npm run tsc",
     "prepare": "npm run build",
     "lint": "eslint .",
-    "test": "node --test",
+    "test": "node --test \"./tests/*.test.js\"",
+    "test:e2e": "mocha e2e/electron-app.test.js",
```

Comment:
Shouldn't e2e tests be part of tests directory?

Reply:
Moved